build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/ClientConfigurationFields.java

```diff
@@ -33,6 +33,7 @@
     String NPM_INSTALL_ARGS = "npm.installArgs";
     String NPM_EXECUTABLE_PATH = "npm.executablePath"; // Path to npm CLI
     String NPM_PATH = "npm.path"; // Path to package.json dir
+    String NPM_MODULE = "npm.module"; // custom module name for npm operation's build-info
```

Comment:
custom module name for npm operation's build-info --> Custom npm module name for the build-info